.github/workflows/push-master-environment.yaml

```diff
@@ -70,8 +70,8 @@ jobs:
           DB_PORT: "5432"
           DB_SCHEMA: ${{ secrets.DB_SCHEMA }}
           EOS_API_ENDPOINT: https://bp.cryptolions.io
-          VIRTUAL_HOST: 0.0.0.0
-          VIRTUAL_PORT: "9090"
+          VIRTUAL_HOST: ${{ secrets.VIRTUAL_HOST }}
+          VIRTUAL_PORT: ${{ secrets.VIRTUAL_PORT }}
```

Comment:
We have to make a balance between what data really needs to be private and the transparency for the developer, does the developer always has access to the secrets?

Reply:
@xavier506

Reply:
I think it is easier to maintain and safer if  non-sensitive env vars values are assigned  in the yaml file